src/utils/AtmosArtifacts.jl

```diff
@@ -82,4 +82,12 @@ function earth_orography_file_path(; context = nothing)
     )
 end
 
+function dyamond_summer_artifact_path(; context = nothing)
+    filename = "DYAMOND_SUMMER_ICS_p98deg.nc"
+    return joinpath(
+        @clima_artifact("DYAMOND_SUMMER_ICS_p98deg", context),
+        filename,
+    )
+end
```

Comment:
Could you add a docstring here with a brief description of this artifact, what it contains, and what's used for?